kombu_redis_priority/transport/redis_priority_async.py

```diff
@@ -668,7 +665,7 @@ def get_table(self, exchange):
         with self.conn_or_acquire() as client:
             values = client.smembers(key)
             if not values:
-                raise InconsistencyError(NO_ROUTE_ERROR.format(exchange, key))
+                return []
```

Comment:
I'm guessing that this code was originally copied from kombu's code so I looked at the reason why `NO_ROUTE_ERROR` doesn't exist anymore and it looks like they think it makes more sense just to return an empty list: https://github.com/celery/kombu/commit/73b10a93464bcf91126c2cf35dbdc096073789df#diff-203e309d100714904255d0af4500dd295e40e4ffedfdc9295ab3b386718ca87eL1000.